src/test/java/mops/ArchitectureRulesTest.java

```diff
@@ -50,31 +47,12 @@ public void onlyAggregateRootsArePublic() {
     @Test
     public void oneAggregateRootPerAggregate() {
         ArchRule oneAggregateRootPerPackage = slices()
-                .matching(".." + MOPS_PERSISTENCE + ".(*)..")
+                .matching(".." + MOPS_PERSISTENCE)
```

Comment:
shouldn't just MOPS_PERSISTENCE be enough ?

Reply:
👍